src/components/ApodContent/ApodContent.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { StoreContext } from '../../utils/store/StoreContext';
+import classes from './ApodContent.module.css';
+
+const ApodContent = () => {
+  const { apiData } = useContext(StoreContext);
+
+  const isImage =
+    Object.keys(apiData).length != 0 &&
+    apiData.url.match(/\w+\.(jpg|jpeg|gif|png|tiff|bmp)$/gi);
+  const isVideo =
+    Object.keys(apiData).length != 0 &&
+    apiData.url.match(
+      /^(?:https?:\/\/)?(?:m\.|www\.)?(?:youtu\.be\/|youtube\.com\/(?:embed\/|v\/|watch\?v=|watch\?.+&v=))((\w|-){11})(?:\S+)?$/
+    );
```

Comment:
I think this could have been done easier with the property `media_type` that the NASA API returns, that property says video or image and that way you can check very easily